exp_to_json.py

```diff
@@ -3,31 +3,69 @@
 
 def exp_to_json(file_path):
 
+    def _reset_switches():
+        global is_job_table, is_documentation, is_module_sched, is_chain_detail
+        global is_job_prompts, is_object_cond
+
+        is_job_table = False
+        is_documentation = False
+        is_chain_detail = False
+        is_module_sched = False
+        is_job_prompts = False
+        is_object_cond = False
+
+    # global switches
+    global is_job_table, is_documentation, is_module_sched, is_chain_detail
+    global is_job_prompts, is_object_cond
+
+    # section index
+    module_sched_idx = chain_idx = -1
+
     exp_json = {
+        'aw_module_sched': [],
+        'so_chain_detail': [],
         'so_job_table': {},
         'so_job_prompts': [],
-        'so_object_cond': []
+        'so_object_cond': {}
     }
+
     check_dict = {
         'B': [],  # program types
         'D': [],  # data type
         'G': [],  # agents
         'K': [],  # application
         'L': [],  # pathes
+        'M': [],  #
+        'n': [],  # notifications
         'O': [],  # logins
+        'P': [],
         'Q': [],  # queues
-        'n': [],  # notifications
-        'R': []   # roles
+        'R': [],  # roles
+        'T': [],
+        'U': []   #
```

Comment:
Some of these elements in `check_dict` don't have comments. Could you add the definitions like you did the other ones, or remove the stray comment markers if you don't know what a letter refers to?